Editor/Window/Components/ProfileSelector.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Linq;
+using Editor.CoreAPI;
+using Editor.Resources.EditorWindow;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace AmazonGameLift.Editor
+{
+    public class ProfileSelector : VisualElement
+    {
+        public new class UxmlFactory : UxmlFactory<ProfileSelector> { }
+
+        private readonly StateManager _stateManager;
+        private readonly TextProvider _textProvider;
+
+        private DropdownField _dropdown => this.Q<DropdownField>("Dropdown");
+        private Label _bucketName => this.Q<Label>("BucketName");
+        private Label _region => this.Q<Label>("Region");
+        private Label _status => this.Q<Label>("BootstrapStatus");
+
+        public ProfileSelector()
+        {
+            var asset = Resources.Load<VisualTreeAsset>("EditorWindow/Components/ProfileSelector");
+            asset.CloneTree(this);
+
+            LocalizeText();
+
+            _stateManager = EditorWindow.GetWindow<GameLiftPlugin>().StateManager;
```

Comment:
I presume there isn't a way to pass this type of data to a component and that's why we're doing this? If that's the case, is a component the best strategy to use here compared to how we're handling our tab pages for example?

Reply:
There isnt a way, no. I personally don't think its that bad a parttern for components to do this, but open to discuss

Reply:
I'm fine moving on from this to unblock this PR.

My reasoning for why this isn't the best pattern is this strongly couples the classes ProfileSelector and GameLiftPlugin which causes problems for maintaining code. The fix here is fairly trivial so if it comes down to it, we can fix it when we need to